apcd-cms/src/apps/exception/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+from apps.exception.views import ExceptionFormView
+from apps.exception.views import ExceptionOtherFormView
+from apps.exception.views import ExceptionThresholdFormView
+
+app_name = 'exception'
+urlpatterns = [
+    path('exception-request/', ExceptionFormView.as_view(), name="index"),
```

Comment:
Can you try these changes to see if it would work having the exception urls in this file?

```suggestion
    path('exception-request/', ExceptionFormView.as_view(), name="index"),
    path('threshold-exception/', ExceptionThresholdFormView.as_view(), name="threshold-exception"),
    path('other-exception/', ExceptionOtherFormView.as_view(), name="other-exception")
```

Reply:
This works for me locally not sure what the issue was on my end originally. Only thing is the path on line 8 needs to be `    path('exception/', ExceptionFormView.as_view(), name="index"),` instead. I'll push these changes up now. Thank you!